bot/guild/channel/channelCreate/autoCreateChannel.module.js

```diff
@@ -0,0 +1,14 @@
+const autoCreateChannel = (server) => {
+  server.channels.create('may-be-scammer', {
+      type: 'GUILD_TEXT',
+      permissionOverwrites: [
+        {
+          id: server.id,
+          allow: ['VIEW_CHANNEL'],
+        },
+      ],
+    },
+  ).then(channel => {channel.send('welcome to use scammer-detector-bot!')})
```

Comment:
Extract the welcome message into a configuration file. So that it could be easy to internationalization.